samples/cpp/stable_diffusion/README.md

```diff
@@ -0,0 +1,46 @@
+# Stable Diffusion C++ Image Generation Pipeline
+
+This example showcases inference of text to image models like Stable Diffusion 1.x, 2.x, LCM. The application doesn't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample features `ov::genai::Text2ImagePipeline` and uses a text prompt as input source.
+
+Users can change the sample code and play with the following generation parameters:
+
+- Change width or height of generated image
+- Generate multiple images per prompt
+- Adjust a number of inference steps
+- Play with [guidance scale](https://huggingface.co/spaces/stabilityai/stable-diffusion/discussions/9) (read [more details](https://arxiv.org/abs/2207.12598))
+- (SD 1.x, 2.x only) Add negative prompt when guidance scale > 1
+
+## Download and convert the models and tokenizers
+
+The `--upgrade-strategy eager` option is needed to ensure `optimum-intel` is upgraded to the latest version.
+
+It's not required to install [../../requirements.txt](../../requirements.txt) for deployment if the model has already been exported.
+
+```sh
+pip install --upgrade-strategy eager -r ../../requirements.txt
+optimum-cli export openvino --model dreamlike-art/dreamlike-anime-1.0 --task stable-diffusion --weight-format fp16 dreamlike_anime_1_0_ov/FP16`
```

Comment:
I see
> Update main README.md

in TODO, but I start reading with the introductory things anyway :)
```suggestion
optimum-cli export openvino --model dreamlike-art/dreamlike-anime-1.0 --task stable-diffusion --weight-format fp16 dreamlike_anime_1_0_ov/FP16
```

Reply:
`main` meant root GenAI readme.
this one is supposed to be ready ;)

fixed